tests/integration/deploy/test_deploy_command.py

```diff
@@ -21,6 +22,8 @@
 CFN_SLEEP = 3
 TIMEOUT = 300
 
+CommandResult = namedtuple("CommandResult", "process stdout stderr")
```

Comment:
No commas? are they not required?

Reply:
Named tuple have two formats, one is array of var names, second is space separated var names. Both work.